### PR TITLE
Add TF version of Ray usage.

### PR DESCRIPTION
**Pull request by coledie**
- [x] Forked repo is synced with upstream -> github shows no code delta outside of the desired.
    https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork
- [x] Tests are passing? https://github.com/microsoft/DeepGNN/blob/main/CONTRIBUTING.md#run-tests
- [x] Changelog and documentation updated.
- [x] PR is labeled using the label menu on the right side.

New Behavior
----------------
Add documentation for example on how to use Ray with Tensorflow.
-> This is a repeat of a previous pull request, this is meant to merge in with the torch ray_usage docs before the migration.

### CODE REVIEW

File: docs/tf/ray_usage.rst [hunk 1]
```diff
@@ -0,0 +1,260 @@
+**************************************************
+Ray Usage Example for Node Classification with GAT
+**************************************************
+
+.. code-block:: python
```
Comment (mortid0) [inline docs/tf/ray_usage.rst:5]: Please add some comments to the code. It is not self-explanatory for someone who starts directly with deepgnn and TF.

**Review by mortid0 — CHANGES_REQUESTED** [review]

File: docs/tf/ray_usage.rst (hunk 1 above)
Comment (coledie) [reply docs/tf/ray_usage.rst:5]: This is a copy of the node class example, only with small changes to make it use Ray.

Comment (mortid0) [reply docs/tf/ray_usage.rst:5]: A person who works with TF doesn't know about a pytorch copy and that they should start with it. Also why should they?

Comment (coledie) [reply docs/tf/ray_usage.rst:5]: That should be for a different PR then, this is meant to be a new version of the tf_node_class that uses Ray. The original version has no documentation either,

**Review by mortid0 — APPROVED** [review]
